gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -44,6 +48,34 @@ ABSL_FLAG(std::string, bhive_csv, "", "Filename of the input BHive CSV file");
 ABSL_FLAG(
     std::string, output_dir, "",
     "Directory containing output files that can be executed by llvm-exegesis");
+ABSL_FLAG(std::string, json_output_dir, "",
+          "Directory containing JSON output files");
+ABSL_FLAG(
+    unsigned, json_split_count, std::numeric_limits<unsigned>::max(),
+    "The number of annotated basic blocks to include in a single JSON file");
+ABSL_FLAG(std::vector<std::string>, output_types,
```

Comment:
isn't there some `enum` based flag that'd self-validate instead?

Reply:
All of the enum flags still need to implement AbslParseFlag and AbslUnparseFlag and I wasn't able to get that working with a vector of enums. I've switched to Ondrej's suggestion and now just have flags for the directories and enable the output if the directory is set.